javascript/classes.json

```diff
@@ -507,10 +507,12 @@
               "version_added": "53"
             },
             "safari": {
-              "version_added": false
+              "version_added": false,
+              "notes": "Added in Safari Technology Preview 109 - Safari Beta 14"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": false,
```

Comment:
Likewise for Safari for iOS (which has slightly different versioning, historically, though in this case the version numbers are the same) https://github.com/mdn/browser-compat-data/blob/master/browsers/safari_ios.json#L161